src/actions/users.ts

```diff
@@ -267,6 +267,19 @@ export function getTotalUsersStats(): ActionFunc {
     });
 }
 
+export function getFilteredUsersStats(options: GetFilteredUsersStatsOpts = {}): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        try {
+            const stats = await Client4.getFilteredUsersStats(options);
+            return {data: stats};
```

Comment:
Should this be stored in redux state somehow? Dispatching a redux action without having the changes reflected in the state is a bit of an antipattern. Only 1/5 on that though since it may not be necessary to store this for a long term or for use in multiple places.